### PR TITLE
[autorestart] address log analyzer issues in autorestart test

### PR DESCRIPTION
**Pull request by yxieca**
### Description of PR
Summary: address log some analyzer issues in autorestart test

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### What is the motivation for this PR?
autorestart test is failing with master image.

#### How did you do it?
- Add new pmon event regrading simulated Y-cable to global ignore list.
- Add new pmon event regrading multi-ASIC to global ignore list.
- Re-organiz the autorestart test ignore list per feature.

Signed-off-by: Ying Xie <ying.xie@microsoft.com>

#### How did you verify/test it?
run autorestart test. With the fix the test passes.

autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|lldp] PASSED                                                                            [  3%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|pmon] PASSED                                                                            [  7%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|sflow] SKIPPED                                                                          [ 10%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|database] SKIPPED                                                                       [ 14%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|snmp] PASSED                                                                            [ 17%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|telemetry] PASSED                                                                       [ 21%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|bgp] PASSED                                                                             [ 25%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|radv] PASSED                                                                            [ 28%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|mgmt-framework] PASSED                                                                  [ 32%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|nat] SKIPPED                                                                            [ 35%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|teamd] PASSED                                                                           [ 39%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|dhcp_relay] PASSED                                                                      [ 42%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|swss] PASSED                                                                            [ 46%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-06|syncd] PASSED                                                                           [ 50%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|lldp] SKIPPED                                                                           [ 53%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|pmon] SKIPPED                                                                           [ 57%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|sflow] SKIPPED                                                                          [ 60%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|database] SKIPPED                                                                       [ 64%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|snmp] SKIPPED                                                                           [ 67%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|telemetry] SKIPPED                                                                      [ 71%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|bgp] SKIPPED                                                                            [ 75%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|radv] SKIPPED                                                                           [ 78%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|mgmt-framework] SKIPPED                                                                 [ 82%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|nat] SKIPPED                                                                            [ 85%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|teamd] SKIPPED                                                                          [ 89%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|dhcp_relay] SKIPPED                                                                     [ 92%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|swss] SKIPPED                                                                           [ 96%]
autorestart/test_container_autorestart.py::test_containers_autorestart[str2-7050cx3-acs-07|syncd] SKIPPED                                                                          [100%]

### CODE REVIEW

**Review by daall — APPROVED** [review]
the swss/syncd messages LGTM, wait on other reviewers for other services

**Comment by yxieca** [comment]
retest vsimage please

**Comment by bingwang-ms** [comment]
Following ERRORs were observed on on A7260. Seems these logs are not covered by patterns. 
```
Dec  2 06:03:17.674319 str-7260cx3-acs-2 ERR swss#buffermgrd: :- doSpeedUpdateTask: Unable to create/update PG profile for port Ethernet256. No PG profile configured for speed 10000 and cable length 300m', 
Dec  2 06:03:17.674319 str-7260cx3-acs-2 ERR swss#buffermgrd: :- doTask: Failed to process invalid entry, drop it',
Dec  2 06:03:17.674319 str-7260cx3-acs-2 ERR swss#buffermgrd: :- doSpeedUpdateTask: Unable to create/update PG profile for port Ethernet260. No PG profile configured for speed 10000 and cable length 300m'
Dec  2 06:03:17.674319 str-7260cx3-acs-2 ERR swss#buffermgrd: :- doTask: Failed to process invalid entry, drop it
```

File: tests/autorestart/test_container_autorestart.py [hunk 1]
```diff
@@ -55,42 +55,42 @@ def ignore_expected_loganalyzer_exception(loganalyzer):
...
     if loganalyzer:
-        loganalyzer.ignore_regex.extend(monit_ignoreRegex)
-        loganalyzer.ignore_regex.extend(swss_ignoreRegex)
-        loganalyzer.ignore_regex.extend(pmon_ignoreRegex)
-        loganalyzer.ignore_regex.extend(syncd_ignoreRegex)
-        loganalyzer.ignore_regex.extend(teamd_ignoreRegex)
-        loganalyzer.ignore_regex.extend(systemd_ignoreRegex)
-        loganalyzer.ignore_regex.extend(kernel_ignoreRegex)
-        loganalyzer.ignore_regex.extend(other_ignoreRegex)
+        loganalyzer.ignore_regex.extend(ignore_regex_dict['common'])
```
Comment (judyjoseph) [inline tests/autorestart/test_container_autorestart.py:91]: A minor comment, will the common set of match regex be added multiple times when ignore_expected_loganalyzer_exception() is called per enum_dut_feature ?

Comment (judyjoseph) [reply tests/autorestart/test_container_autorestart.py:91]: Ignore my comment Ying .. I think it should be ok as it is parameterized and will be invoked now separately per enum_dut_feature.

**Review by judyjoseph — APPROVED** [review]
LTGM

File: tests/autorestart/test_container_autorestart.py (hunk 1 above)
Comment (yxieca) [reply tests/autorestart/test_container_autorestart.py:91]: I think we are fine. the loganalyzer is setup per testcase.

**Pull request merged**